Vagrantfile

```diff
@@ -168,6 +168,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       app.vm.synced_folder "src/nyc_trees/apps", "/opt/app/apps"
     else
       app.vm.synced_folder "src/nyc_trees", "/opt/app/"
+      app.vm.synced_folder "src/tiler/sql", "/opt/tiler/sql/"
```

Comment:
I think this needs a comment to explain _why_ we're syncing the tiler SQL into the app